server.js

```diff
@@ -44,7 +44,7 @@ var html = ({ title, meta, link }, data, body) => {
 
 app.use((req, res, next) => {
   var actingUrl = req.url.replace(/\/(.*)\/$/, '/$1')
-  const internalIPs = process.env.PRIVILEGED_IP_LIST
+  const internalIPs = process.env.PRIVILEGED_IP_LIST || '';
```

Comment:
Rather than force users to provide the environment variable I thought it made sense to default to an empty string